addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -835,16 +835,23 @@ var VideoWidget = MediaWidget.extend({
         if (this.media) {
             var $media = $(this.media);
             var src = $media.data('oe-expression') || $media.data('src') || '';
+            var opacity = $media.data('opacity') || '';
+            var background = $media.data('background') || '';
+            var container = $media.data('container') || '';
             this.$('textarea#o_video_text').val(src);
 
             this.$('input#o_video_autoplay').prop('checked', src.indexOf('autoplay=1') >= 0);
             this.$('input#o_video_hide_controls').prop('checked', src.indexOf('controls=0') >= 0);
             this.$('input#o_video_loop').prop('checked', src.indexOf('loop=1') >= 0);
             this.$('input#o_video_hide_fullscreen').prop('checked', src.indexOf('fs=0') >= 0);
             this.$('input#o_video_hide_yt_logo').prop('checked', src.indexOf('modestbranding=1') >= 0);
+            this.$('input#o_video_muted').prop('checked', src.indexOf('mute=1') >=0);
```

Comment:
Missing space around operators, check your linter, it should have told you that mistake :wink:
https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines